scripts/dts/extract_dts_includes.py

```diff
@@ -275,6 +275,9 @@ def extract_node_include_info(reduced, root_node_address, sub_node_address,
         except KeyError:
             pass
 
+    # Extract status
+    if 'status' in y_node['properties'].keys():
```

Comment:
default.extract for "status" will be called in extract_property() anyway - why does it have to be called here?

Reply:
There is currently no binding associated to 'status' so 'status' property is not treated by extract_property. So we could add yaml bindings for 'status' in every yaml binding. Side effect would be #define generation for each status, which is not requested today. Then it would require additional treatment to inject in edts w/o generating a define.

So I chose to add this treatment here, which is a bit off track I admit but since we're not clear yet on the use of status property, I did it the cheap way.

Reply:
> There is currently no binding associated to 'status' so 'status' property ...

ok. Maybe still the "better" way to include it at least in the generic device bindings like i2c.yaml, i2c-device.yaml, uart.yaml, ... This may already cure a lot of device bindings that include the generic ones.

> Then it would require additional treatment to inject in edts w/o generating a define.

Define creation in default.py can be avoided by creating a specific extract/status.py and calling status.extract() from extract_property(). That was my original thinking when I suggested to create a status.py. 

As long as not all bindings are changed to include the status property status.extract() may be called here and in  extract_property(). The double call does not hurt and after all bindings are changed the call can be removed here. - Not really nice and definitely needs a comment to explain the removal.